common-rest/src/main/java/org/uniprot/api/rest/validation/CustomConstraintGroup.java

```diff
@@ -0,0 +1,3 @@
+package org.uniprot.api.rest.validation;
+
+public interface CustomConstraintGroup {}
```

Comment:
Can we have a more specific name that shows the purpose? 
Ex: DownloadSubmissionVerificationGroup

Reply:
The reason for choosing that name was that the Group can be used for any other validation where we need custom behaviour.